surround360_camera_ctl/README.md

```diff
@@ -106,7 +106,7 @@ After installing all of the dependencies as described above, run:
 To test that compilation is successful, run:
 
 ```
-  ./bin/CameraControl -numcams 17 -raw -nbits 8 -shutter 20 -gain 0 -debug
+  sudo ./bin/CameraControl -numcams 17 -raw -nbits 8 -shutter 20 -gain 0 -debug
```

Comment:
ok, that can be used as a workaround in case you didn't run setup properly (which one should do) or as an alternative create the directory with sudo, etc. (what setup does) manually ...

a pull request where that path is not hardwired but configurable would be the much cleaner solution IMHO ;)